jcl/src/jdk.management/share/classes/com/ibm/lang/management/OperatingSystemMXBean.java

```diff
@@ -41,24 +40,26 @@
  */
 public interface OperatingSystemMXBean extends com.sun.management.OperatingSystemMXBean {
 
+	/*[IF JAVA_SPEC_VERSION < 14] - inherit the default method in Java 14+ */
```

Comment:
It seems it can be inherited in Java 8 & 11 as well. Same for getSystemCpuLoad() and getFreePhysicalMemorySize()

Reply:
Inheriting the declarations of `getFreePhysicalMemorySize()` and `getSystemCpuLoad()` will eliminate the javadoc comments specific to z/OS: Are you ok with that?

Reply:
The same problem exists for 14+. How about adding a javadoc comment for the class to mention these APIs are unsupported on z/OS, and then they can be removed for all versions.